blankiety_galantow/core/room.py

```diff
@@ -22,10 +23,21 @@ def __init__(self, name):
         self.settings.name = name
         self.admin = None
 
+    def __del__(self):
+        self.timer_cancel()
```

Comment:
Dobra, może trochę Ci źle powiedziałem, ale sam nie wiedziałem, czy wszystko będzie działało.
Nie potrzebujemy tej metody.
możemy zrobić:
```python
def __del__(self):
        self.game_master.timer.cancel()
```
Nie musimy jednak robić tego try except bo już w timer_start to mamy:
```python
async def timer_start(self):
...
        if self.timer is not None:
            self.timer.cancel()
            try:
                await self.timer.task
            except asyncio.CancelledError:
                pass
            except KickException:
                raise
...
```
Także kod się jeszcze uprości i nie będzie tego błędu:
![error](https://user-images.githubusercontent.com/37443550/83942034-898e1080-a7f0-11ea-8b6c-e70317347ef0.png)